octobot_trading/personal_data/portfolios/portfolio_value_holder.py

```diff
@@ -256,7 +279,51 @@ def convert_currency_value_using_last_prices(self, quantity, current_currency, t
             pass
         except KeyError:
             pass
-        raise errors.MissingPriceDataError(f"no price data to evaluate {current_currency} price in {target_currency}")
+        raise errors.MissingPriceDataError(
+            f"no price data to evaluate {current_currency} price in {target_currency}")
+
+    def _try_converting_with_multiple_pairs(
+        self, currency: str, quantity: decimal.Decimal) -> decimal.Decimal:
+        # try with two pairs
+        # for example: 
+        # currency: ETH - ref market: USDT
+        #                   BTC/ETH      ->    BTC/USDT
+        # first covert ETH -> BTC and then BTC -> USDT
+        for symbol in symbol_data.get_config_symbols(
+            self.portfolio_manager.exchange_manager.config, True
+        ):
+            if symbol.startswith(currency):
+                first_ref_market = symbol_util.parse_symbol(symbol).quote
+                second_symbol = (
+                    f"{first_ref_market}/{self.portfolio_manager.reference_market}"
```

Comment:
You should prefer to use https://github.com/Drakkar-Software/OctoBot-Commons/blob/master/octobot_commons/symbols/symbol_util.py.

Reply:
I changed it, now it uses symbol util properly